assets/app/view/game/map_controls.rb

```diff
@@ -81,13 +87,15 @@ def route_controls
           operator = all_operators.find { |o| o.name == operator_name }
           if operator
             store(:historical_routes, generate_last_route(operator))
+            store(:historical_laid_hexes, last_laid_hexes(operator))
```

Comment:
you need to add skip: true to both of these so you don't rerender twice

Reply:
Good catch. Fixed.